components/GrantApplicationForm.tsx

```diff
@@ -73,11 +73,12 @@ export default function ApplicationForm() {
           {...register('main_focus', { required: true })}
         >
           <option value="">(Choose One)</option>
-          <option value="layer1">Bitcoin / Layer1</option>
-          <option value="layer2">Lightning / Layer2</option>
           <option value="core">Bitcoin Core</option>
           <option value="education">Education</option>
-          <option value="nostr">nostr</option>
+          <option value="layer1">Layer1 / Bitcoin</option>
```

Comment:
```suggestion
          <option value="layer1">Layer 1 / Bitcoin</option>
```

Reply:
Just adding the edit suggestion by @ecurrencyhodler directly into the file here, and elsewhere regarding spacing between "Layer" and the layer number. Otherwise the main order focus list looks good, starting at Bitcoin Core >> Layer 1 >> Layer 2 >> Layer 3 🚀

Reply:
I believe the lack of spacing is intentional.